odxtools/table.py

```diff
@@ -143,9 +157,13 @@ def read_table_from_odx(et_element):
         read_table_row_from_odx(el) for el in et_element.iterfind("TABLE-ROW")
     ]
 
+    table_row_refs = [
+        TableRowRef(el.get('ID-REF')) for el in et_element.iterfind("TABLE-ROW-REF")
+    ]
+
     table = Table(
         key_dop_ref=key_dop_ref,
-        table_rows=table_rows,
+        table_rows=table_rows + table_row_refs,
```

Comment:
I think it would be better to add a `table_row_refs` attribute to the table class, rename the current `table_rows` attribute to `_local_table_rows` (or similar) and introduce a new `table_rows` attribute which returns the union of the resolved table row references and the rows directly specified for the table. this is analogous to how inheritance is handled, cf. e.g. `ecu._local_services` vs `ecu.services`.